server/odc-service/src/main/java/com/oceanbase/odc/service/task/schedule/JobScheduler.java

```diff
@@ -45,15 +47,24 @@ public interface JobScheduler {
      */
     void cancelJob(Long jobId) throws JobException;
 
+    /**
+     * modify job parameters, new parameters will be delivery to executor if job is running
+     *
+     * @param jobId job id
+     * @param jobParameters job parameters
+     * @throws JobException throw JobException if job is terminated
+     */
+    void modifyJobParameters(Long jobId, Consumer<Map<String, String>> jobParameters) throws JobException;
```

Comment:
The input parameters of the method are a bit complicated.

Reply:
modify method parameter, remove consumer